### ISSUE TITLE
discernParser is documented incorrectly

### ISSUE THREAD

**Issue by aubergene** [mhkeller/indian-ocean#67]
I was expecting to be able to do this

```js
var parser = io.discernParser('blah.csv')
parser('name,price\nApple,1513\nPear,300')
```

**Comment by mhkeller** [comment]
Ya that's a good catch. It's [documented correctly](http://mhkeller.github.io/indian-ocean/#helpers/discernparser) as returning a function that takes a file path but this would make more sense to rename and then separately create functions that expose the parser.

**Comment by aubergene** [comment]
Yeah, I think it would be good, then it easy to use it in situations where you've read the file by some other mechanism already.

**Comment by aubergene** [comment]
Wait, it's actually correct, it does return a parser, and works exactly I wanted, so it's just documented incorrectly, hoorah!

**Comment by mhkeller** [comment]
Hooray!

**Comment by mhkeller** [comment]
FYI i think the usage of this function is a bit bonkers since it has these weird two optional arguments whether it's a file path or a delimiter. I was planning on refactoring it later on when times seem worthy of a breaking change. Something like a first argument as a string and on option options object with `delimter: true` if that string should be treated as a delimiter

```js
io.discernParser(strToTest[,{delimier: true}])
```

Issue: https://github.com/mhkeller/indian-ocean/issues/63

**Comment by mhkeller** [comment]
This is fixed via 2c4ec09d44295412bda3ff9803bdc9db410bf1e7

**Comment by mhkeller** [comment]
Published in 3.0.0
